### PR TITLE
Add bedrock2 instructions to README

### PR DESCRIPTION
**Pull request by JasonGross**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,27 +1,36 @@
...
 Building
------
+--------
 [![CI (Coq)](https://github.com/mit-plv/fiat-crypto/workflows/CI%20(Coq)/badge.svg)](https://github.com/mit-plv/fiat-crypto/actions?query=workflow%3A%22CI+%28Coq%29%22+branch%3Amaster)
 
-This repository requires Coq 8.9 or later. Note that if you install Coq from Ubuntu aptitude packages, you need `libcoq-ocaml-dev` in addition to `coq`.
+This repository requires [Coq](https://coq.inria.fr/) [8.9](https://github.com/coq/coq/releases/tag/V8.9.0) or later.
+Note that if you install Coq from Ubuntu aptitude packages, you need `libcoq-ocaml-dev` in addition to `coq`.
+If you want to build the bedrock2 code, you need [Coq 8.10](https://github.com/coq/coq/releases/tag/V8.10.0) or later (otherwise you can pass `SKIP_BEDROCK2=1` to `make`).
+We suggest downloading [the latest version of Coq](https://coq.inria.fr/download).
```
Comment (andres-erbsen) [inline README.md:11]: Please link to https://github.com/coq/coq/wiki#coq-installation instead

File: README.md [hunk 2]
```diff
@@ -31,13 +40,13 @@ A collection of C files for popular curves can be made with
...
-The C files will appear in the top-level directory.
+The C files will appear in [`fiat-c/src/`](./fiat-c/src/).
 
 Just the compilers generating these C files can be made with
 
-    make standalone
+    make standalone-ocaml
 
-or `make standalone-haskell` or `make standalone-ocaml` for binaries generated with just one compiler.
+or `make standalone-haskell` for binaries generated with Haskell, or `make standalone` for both the Haskell and OCaml binaries.
```
Comment (andres-erbsen) [inline README.md:49]: "compiler binaries"?

File: README.md [hunk 3]
```diff
@@ -60,6 +69,58 @@ Here are some examples of ways to invoke the binaries (from the directories that
 
 You can find more examples in the `Makefile`.
 
+Usage (Generating Bedrock2 Files)
+---------------------------------
+
+The Coq development builds binary compilers that generate code using some implementation strategy.
+The parameters (modulus, hardware multiplication input bitwidth, etc.) are are specified on the command line of the compiler.
+The generated bedrock2/C code is written to standard output.
```
Comment (andres-erbsen) [inline README.md:77]: Perhaps a sentence to explain that this is fiat-crypto targetting bedrock2, and then printing C code using the bedrock2 C backend?

File: README.md (hunk 1 above)
Comment (JasonGross) [reply README.md:11]:
```suggestion
We suggest downloading [the latest version of Coq](https://github.com/coq/coq/wiki#coq-installation).
```

File: README.md (hunk 2 above)
Comment (JasonGross) [reply README.md:49]:
```suggestion
or `make standalone-haskell` for compiler binaries generated with Haskell, or `make standalone` for both the Haskell and OCaml compiler binaries.
```

File: README.md (hunk 3 above)
Comment (JasonGross) [reply README.md:77]:
```suggestion
The generated bedrock2 code is then written to standard output using the bedrock2 C backend.
```

Comment (JasonGross) [reply README.md]: Is this good?

Comment (andres-erbsen) [reply README.md]: Yes

File: README.md [hunk 4]
```diff
@@ -60,6 +69,58 @@ Here are some examples of ways to invoke the binaries (from the directories that
 
 You can find more examples in the `Makefile`.
 
+Usage (Generating Bedrock2 Files)
+---------------------------------
+
```
Comment (jadephilipoom) [inline README.md:74]:
```suggestion
Output is supported to the research language [bedrock2](https://github.com/mit-plv/bedrock2).
```
I'd also like to explain something like "the C code for bedrock2 is really a C wrapper for RISC-V code which is why it's weird" but I'm not sure how to phrase that. @andres-erbsen?

File: README.md [hunk 5]
```diff
@@ -1,27 +1,36 @@
...
 
-To build (if your COQPATH variable is empty):
+from inside the repository.
+Then run:
 
-       make
+    make
 
-To build:
+You can check out [our CI](https://github.com/mit-plv/fiat-crypto/actions?query=branch%3Amaster+workflow%3A%22CI+%28Coq%29%22) to see how long the build should take; as of the last update to this line in the README, it takes about 1h10m to run `make -j2` on Coq 8.11.1.
```
[the remark this answers is not in the document]
Comment (andres-erbsen) [reply README.md:26]: I think it is worth mentioning for the order of magnitude (i.e., not seconds) but yeah we have no claim for it being reproducible

File: README.md (hunk 4 above)
Comment (andres-erbsen) [reply README.md:74]: Mh. I am not sure how. Perhaps "the main target of bedrock2 is risc-v; the C code produced by it is unlikely to be idiomatic"?

File: README.md (hunk 5 above)
[the remark this answers is not in the document]
Comment (JasonGross) [reply README.md:26]: I think it's worth it for calibration.  It's probably not going to finish in 5 minutes regardless of machine, and it's probably not going to take 12 hours, unless the machine is extremely slow.

File: README.md (hunk 4 above)
Comment (JasonGross) [reply README.md]: If one of you wants to make a suggestion, I'm happy to include it.  Otherwise we should probably merge this all soon for @achlipala ?

**Comment by achlipala** [comment]
I will only add that, yes, something concrete for me to poke tomorrow morning would be great!

File: README.md (hunk 5 above)
[the remark this answers is not in the document]
Comment (andres-erbsen) [reply README.md:26]: I'm fine with it as is. if you want to improve, you could name the machine for which this timing was, even if the best we can do is "on GitHub CI". I think it's better merged than not merged either way.

**Comment by andres-erbsen** [comment]
I'm merging this because it's timely and no blockers have been identified after two reviews. it's a readme-only change so the CI failure can't be critical.

**Comment by andres-erbsen** [comment]
"Pull requests that have a failing status can’t be merged on a phone." ugh GitHub I'm not 5.

**Comment by JasonGross** [comment]
I think the "request desktop site" checkbox will convince GitHub that you're not 5

**Comment by andres-erbsen** [comment]
It did not

**Pull request merged**